webservices/ruby/style_guide.md

```diff
@@ -0,0 +1,492 @@
+# Ruby Code Style Guidelines.
+#### Each bullet links to an example of both the incorrect and correct pattern(s) for that particlar rule.
+
+## [General](#general)
+* **[Prefer double quotes for strings](#prefer-double-quotes-for-strings)**
+* **[Don't use spaces around square brackets](#dont-use-spaces-around-square-brackets)**
+* **[Use spaces around curly braces](#use-spaces-around-curly-braces)**
+* **[Use a trailing comma after the last item in a multi-line list](#use-a-trailing-comma-after-the-last-item-in-a-multi-line-list)**
+* **[Avoid explicit return statements](#avoid-explicit-return-statements)**
+* **[Avoid multiple assignments per line](#avoid-multiple-assignments-per-line)**
+* **[Prefer && and || over AND and OR](#prefer--and--over-and-and-or)**
+* **[Prefer ! over not](Prefer--over-not)**
+* **[Use %{} for single-line strings needing interpolation and double-quotes](use--for-single-line-strings-needing-interpolation-and-double-quotes)**
+* **[Avoid using semicolons](#avoid-using-semicolons)**
+* **[Avoid organizational comments](#avoid-organizational-comments)**
+* **[Use CamelCase for classes and modules, snake_case for variables, methods, and file names, SCREAMING_SNAKE_CASE for constants.](#use-camelcase-for-classes-and-modules-snake_case-for-variables-methods-and-file-names-screaming_snake_case-for-constants)**
+
+## [Conditionals](#conditionals)
+* **[Avoid lines that end with conditionals](#avoid-lines-that-end-with-conditionals)**
```

Comment:
I haven't seen this one before in a style guide.  I think it should be more specific.  A simple line ending in a conditional can often avoid complicated nesting logic.  I agree that a really long line that ends in an `if` or `unless` clause could be complicated to read, but a simple one like `a = 1 if a.nil?` isn't so bad.

Reply:
Agreed, I was a little torn on this rule for the same reason so I've adjusted it to be more lenient.